README.md

```diff
@@ -1,83 +1,118 @@
-Scel
-====
+# scel - sclang-mode for emacs
 
 SuperCollider/Emacs interface
 
+## Installation
 
-Installation requirements
--------------------------
+There are 3 options for installation:
 
-For the HTML help system, you will need emacs-w3m support.
+1. Using Emacs' and SuperCollider's own package managers (recommended)
+2. From debian package `supercollider-emacs`
+3. From source
 
+Option #1 is the best cross-platform option, and is recommended. Whatever option
+you choose, you should only install from a single source.
 
-Installation (default)
-----------------------
+### Install Option 1: Native Packages
 
-By default emacs-lisp files are installed in
+The repository contains two subprojects. `/el` contains the emacs-lisp
+implementation. `/scide_scel` contains the SuperCollider code required to
+implement the Emacs interface. Emacs and SuperCollider each have their own
+package managers, so it is required to install each half separately.
 
-`$prefix/share/emacs/site-lisp`
+#### Installing scel quark
 
-SuperCollider files are put in
+The `scel` Quark is required for emacs to communicate with sclang. Evaluate this
+code in the SuperCollider GUI:
 
-`$prefix/share/SuperCollider/Extensions/scide_scel`
+``` supercollider
+Quarks.install("https://github.com/supercollider/scel");
+```
+
+The scel repository will be downloaded to your local file system and the path
+will be added to your default `sclang_conf.yaml` file. If you want to change the
+yaml file used by your emacs sessions you can customize the
+`sclang-library-configuration-file` variable, but make sure that file loads 
+the scel quark.
 
+#### Installing the emacs mode
 
-The only thing you need to do is loading the sclang interface in your `~/.emacs`:
+Using [straight.el](https://github.com/raxod502/straight.el), put the following
+in your init file:
 
+``` emacs-lisp
+;; in ~/.emacs
+(straight-use-package
+ '(sclang :type git 
+          :host github 
+          :repo "supercollider/scel" 
+          :files ("el/*.el")))
 ```
+
+Or download the repo directly to your user config directory
+
+``` shell
+git clone https://github.com/supercollider/scel.git ~/.emacs.d/scel
+```
+
+``` emacs-lisp
+;; in ~/.emacs
+(add-to-list 'load-path "~/.emacs.d/scel/el/")
 (require 'sclang)
 ```
 
-For the HTML help system to fully function also add
+#### On macOS
+
+If `sclang` executable is not on your path, you may need to add it to your
+exec-path.
+
+``` emacs-lisp
+(setq exec-path (append exec-path '("/Applications/SuperCollider.app/Contents/MacOS/")))
 ```
-(require 'w3m)
+
+### Install Option 2: Debian package
+
+There is a debian package which provides emacs integration called
+`supercollider-emacs`. Option #1 will likely be more recent, but 
+if you prefer you can install the package with:
+
+``` shell
+sudo apt install supercollider-emacs
 ```
 
+### Install Option 3: Installing from source
```

Comment:
If we make the installing from source deprecated/obsolete, we should indicate it in the readme. Probably for later, not in this PR, but just mentioning it for the future.

Reply:
I had a similar thought, but I didn't want to presume too much. Perhaps after we are listed on MELPA?